pkg/network/driver/common.go

```diff
@@ -164,6 +166,16 @@ func (h *NetworkUtilsHandler) ConfigureIpForwarding(proto iptables.Protocol) err
 	return err
 }
 
+func (h *NetworkUtilsHandler) ConfigurePingGroupRange() error {
+	err := sysctl.New().SetSysctl(sysctl.PingGroupRange, "0 2147483647")
```

Comment:
do we need this broad of a range? or can it just be group id 107?

Reply:
Good question, I will try it. Thanks!